includes/parser/Parser.php

```diff
@@ -1264,6 +1264,11 @@ function doMagicLinks( $text ) {
 					[0-9Xx]                 # check digit
 					\b)
 			)!xu', array( &$this, 'magicLinkCallback' ), $text );
+
+		if ( preg_last_error() !== 0 ) {
+			\Wikia\Logger\WikiaLogger::instance()->error( 'PCRE error', [ 'preg_last_error' => preg_last_error() ] );
```

Comment:
Well, it would, but I'm not interested yet in having full backtrace (especially that I know where the logger is called). If the idea is that every error should log an Exception then probably it should implemented in the logger itself.